app/customer.py

```diff
@@ -0,0 +1,16 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Customer:
+    name: str
+    product_cart: dict
+    location: list
+    money: float
+    car: callable
+    shop_info: list
+
+    def trac_distance(self, shop_location: list) -> float:

```

Comment:
typo: trac_distance -> track_distance

Reply:
Also, add prefix get_
for all methods and function the name must be a verb. for properties and attributes noun